### PR TITLE
add support of tab penl lazy render(#11022)

### LINKED ISSUE #11022: [Feature Request] Tabs lazy loading
### Existing Component
Yes

### Component Name
Tabs

### Description
Hi,

Is there a way to lazy load tab contents?

Since tabs may contain entire sections of the app, this would decrease load time drastically.
Of course, is it possible to manage this at the contents level, but having a native solution would require less boilerplate.

Thanks!

### PR DESCRIPTION
**Pull request by Kingwl**
Please make sure these boxes are checked before submitting your PR, thank you!

Closes #11022

* [ x ] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md)).
* [ x ] Make sure you are merging your commits to `dev` branch.
* [ x ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by Leopoldthecoder** [comment]
谢大佬。准备放在 2.4 里。

**Comment by Kingwl** [comment]
2.4啥时候发布呀。。。

**Comment by Leopoldthecoder** [comment]
最快下周

**Comment by jikkai** [comment]
@Kingwl 大佬求补个 `tab-pane.d.ts` 和西班牙语文档吧 🌚

File: examples/docs/en-US/tabs.md [hunk 1]
```diff
@@ -393,3 +393,4 @@ Only card type Tabs support addable & closeable.
 | disabled | whether Tab is disabled | boolean | — | false |
 | name      | identifier corresponding to the activeName of Tabs, representing the alias of the tab-pane | string | — | ordinal number of the tab-pane in the sequence, e.g. the first tab-pane is '1' |
 | closable  | whether Tab is closable | boolean   | — |  false  |
+| lazy  | whether Tab is lazy to render   | boolean   | — |  false  |
```
[the remark this answers is not in the document]
Comment (jikkai) [reply examples/docs/en-US/tabs.md]: ok 🧐

**Comment by Kingwl** [comment]
大佬们动作好快

**Pull request merged**